app/app.go

```diff
@@ -1224,12 +1262,58 @@ func (app *App) setAnteHandler(txConfig client.TxConfig, wasmConfig wasmtypes.Wa
 			IBCKeeper:         app.IBCKeeper,
 			WasmConfig:        &wasmConfig,
 			TXCounterStoreKey: txCounterStoreKey,
+
+			TxEncoder:     txConfig.TxEncoder(),
+			BuilderKeeper: app.BuilderKeeper,
+			Mempool:       mempool,
 		},
 	)
 	if err != nil {
 		panic(fmt.Errorf("failed to create AnteHandler: %s", err))
 	}
-	app.SetAnteHandler(anteHandler)
+	return anteHandler
+}
+
+// SetupProposalHandler sets the gov proposal handler for skip-pob
```

Comment:
nit: This comment is incorrect. The proposal handlers set here are the ABCI++ Prepare / Process proposal handlers. These methods perform custom logic when validator nodes are instructed to create proposals, would update comment accordingly